### ISSUE TITLE
After activating ‘Contoso.png’ clickable controls, keyboard focus not going first interactive element of opened ‘Media file’ popup. 

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#116]
User Experience:

Users who rely on keyboard for navigating between elements would find it confusing when focus moves is not logical after activating any control.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode.

4. Navigate to the Left nav and select Media button.

5. Navigate to the "Media" window and select any media content.

6. Navigate to the various controls of the "Media file" dialog box.

7. Verify the issue.

 

Actual Result:

After activating ‘Contoso.png’ clickable controls, keyboard focus not going first interactive element of opened ‘Media file’ popup.

 

Expected Result:

After activating ‘Contoso.png’ clickable controls, keyboard focus should move on first interactive element as ‘Name’ edit control of opened ‘Media file’ popup.

 

Note:

Similar issue exist with Chrome.

 

MAS Reference:

MAS 3.2.1 - On Focus (06)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
